osf/models/cedar_metadata.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from osf.utils.datetime_aware_jsonfield import DateTimeAwareJSONField
+from .base import BaseModel, ObjectIDMixin
+
+
+class CedarMetadataTemplate(ObjectIDMixin, BaseModel):
+    title = models.CharField(max_length=255)
+    template = DateTimeAwareJSONField(default=dict)
+    active = models.BooleanField(default=True)
+    template_version = models.PositiveIntegerField()
+
+    class Meta:
+        unique_together = ('title', 'template_version')
+
+    def __unicode__(self):
+        return f'({self.title}, version {self.tempate_version})'
```

Comment:
I am not familiar with metadata. Wondering if title can change for the same metadata. Will it be better if we have template `name` and `version` as key so we can freely change the `title?

Reply:
Whether it is `name` or `title` I don't think it is suppose to change? I think we should get that only from CEDAR instead of customizing them on our end. Also we can customize that on CEDAR if it comes to that.

Reply:
Ah, I see, make sense. I also updated the ticket description slightly, and will merge after checking if migration is up-to-date.

Reply:
Thanks. Also I'd say don't worry about the migrations? We'll have to run `makemigrations` again after all the models are created since we probably don't want to have more than 1 migration files generated for this project.